app/views/all/local_authorities/projects/show.html.erb

```diff
@@ -25,7 +26,8 @@
             <tr class="govuk-table__row">
               <td class="govuk-table__header govuk-table__cell"><%= project.establishment.name %></td>
               <td class="govuk-table__cell"><%= project.urn %></td>
-              <td class="govuk-table__cell"><%= project.significant_date.to_formatted_s(:govuk) %></td>
+              <td class="govuk-table__cell"><%= project.significant_date.to_formatted_s(:significant_date) %></td>
+              <td class="govuk-table__cell"><%= project.type %></td>
```

Comment:
This will output Conversion::Project or Transfer::Project, we want to show just Conversion or Transfer.

Have a look at Project#type_locale - this will return "conversion_project" or "transfer_project" and you can use that to build up a locale key for the correct value.